src/pods/edit-table/components/commands/commands.component.tsx

```diff
@@ -29,6 +41,30 @@ export const Commands: React.FC<Props> = (props: Props) => {
         icon={<RemoveIcon />}
         onClick={() => onDeleteField(field.id)}
       />
+      {fields[fields.length - 1].id !== field.id ? (
+        <CommandIconButton
+          icon={<DownIcon />}
+          onClick={() => onMoveDownField(field.id)}
+        />
+      ) : (
+        <CommandIconButton
+          icon={<DownIcon />}
+          onClick={() => onMoveDownField(field.id)}
+          disabled={true}
+        />
+      )}
```

Comment:
MMm.. I think we could write this as:

```tsx
        <CommandIconButton
          icon={<DownIcon />}
          onClick={() => onMoveDownField(field.id)}
          disabled={fields[fields.length - 1].id !== field.id }
        />
```

So it would be something like:

```diff
-      {fields[fields.length - 1].id !== field.id ? (
        <CommandIconButton
          icon={<DownIcon />}
          onClick={() => onMoveDownField(field.id)}
+       disabled={fields[fields.length - 1].id !== field.id }
        />
-      ) : (
-        <CommandIconButton
-          icon={<DownIcon />}
-          onClick={() => onMoveDownField(field.id)}
-          disabled={true}
-        />
-      )}
```

Reply:
And as additional enhancement we can add a small helper function (even could be moved to business and unit tested):

```ts
const isLasItemInArray = (fields : FieldVm[], fieldId :  GUID) : boolean =>
  fields[fields.length - 1].id !== fieldId
```

The the code would be a bit easier to read:

```tsx
<CommandIconButton
          icon={<DownIcon />}
          onClick={() => onMoveDownField(field.id)}
          disabled={isLasItemInArray(fields, field.id)}
 />
```

Reply:
It's fixed